src/RequestBuilder/ItemPropertiesModifierRequestBuilder.php

```diff
@@ -7,10 +7,8 @@
 use Lmc\Matej\Model\Command\ItemPropertySetup;
 use Lmc\Matej\Model\Request;
 
-class ItemPropertiesSetupRequestBuilder extends AbstractRequestBuilder
+class ItemPropertiesModifierRequestBuilder extends ItemPropertiesGetterRequestBuilder
```

Comment:
If the inheritance is there just because of the endpoint path, I'd not introduce it at all and rather have the endpoint path copied on two places. "Setup request builder" is from my POV not specific version of "Get request builder". It would only make sense IMHO to have `AbstractItemPropertiesRequestBuilder` with two childs, but would be a bit overengineered :).